src/sidebar/components/ThreadList.tsx

```diff
@@ -209,6 +269,11 @@ export default function ThreadList({ threads }: ThreadListProps) {
           id={child.id}
           key={child.id}
         >
+          {headings.get(child) && (
+            <h3 className="text-md text-grey-7 font-bold pt-3 pb-2">
```

Comment:
`text-grey-7` isn't a standard "text color" (standard colors could be applied with `text-color-text` or `text-color-text-light`), but I concede that `text-color` has visual-weight issues (it's too heavy in this context and competes with the tab labels) and `text-color-text-light` doesn't cut the mustard, contrast-wise, on the sidebar-background grey. In a broader sense, I'd like to see if we could improve the design of these headings, especially if/after we consider a thread-virtualization replacement, which would give us more breathing room for making adaptations. So, I'm OK with leaving `text-grey-7` here but I think we should comment it for later review.

Reply:
> I'd like to see if we could improve the design of these headings, especially if/after we consider a thread-virtualization replacement, which would give us more breathing room for making adaptations.

OK, if I get this landed do you want to have a play around with the styling?